kore/CHANGELOG.md

```diff
@@ -14,6 +14,19 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## [0.48.0.0] - 2021-06-16
+
+### Added
+
+- Adds support for Z3's `:rlimit` with `--smt-rlimit` CL option (#2407).
+- Enables building with `-eventlog` by default (#2658).
+- Adds option `--not-matching-children` to `kore-prof` (#2631).
+
+### Fixed
+
+- The `SideCondition.andCondition` performance regression is fixed (#2454).
+- Decoding bytes is improved (fixes added in relation to #2591).
+
```

Comment:
I am not sure if this should mention anything about the work on the Predicate simplifier, which has been partially completed.

Reply:
No, I think that isn't a user-facing change by itself. When the related bugs are fixed, we can report those.